src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -153,6 +155,7 @@ protected OptionParser buildOptionParser() {
 		parser.accepts( "kafka_version", "use kafka 0.8, 0.9, 0.10, 0.10.1, or 0.10.2 producer (default 0.9)");
 
 		parser.accepts( "kinesis_stream", "kinesis stream name").withOptionalArg();
+		parser.accepts("sqs_queue_uri", "SQS Queue uri").withOptionalArg();
```

Comment:
The value for `sqs_queue_uri` needs to be supplied? In that case, you'll need to use `withRequiredArg` here.

Reply:
resolved